app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,19 @@
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+
+    <%= f.email_field :email, autofocus: true, placeholder: 'Email', autocomplete: "email" %>
+  </div><br/>
+
+  <div class="field">
+
+    <%= f.password_field :password, placeholder: 'Password', autocomplete: "current-password" %>
+  </div><br/>
+
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
```

Comment:
- [ ] As requested earlier, kindly follow the given design for the login and sign up page as this is required

Your design
![image](https://user-images.githubusercontent.com/81932478/216198759-b7003343-af74-4959-8da4-3c4be04ae07e.png)

Given wireframe
![image](https://user-images.githubusercontent.com/81932478/216198915-a9061198-bd69-4752-b41c-77c21a3501f7.png)